Resources/doc/index.md

```diff
@@ -69,6 +63,34 @@ class AppKernel extends Kernel
  > }
  > ```
 
+Usage
+-----
+
+### Mapping Objects
+
+Refer to the [Hateoas documentation](http://github.com/willdurand/Hateoas) to
+find out how to map your objects.
+
+### Serializing objects
+
+The BazingaHateoasBundle transparently hooks into the JMS serializer, there
+are no special considerations:
+
+````php
+// My/Controller.php
+
+class SomeController extends Controller
+{
+    public function resourceAction(Request $request)
+    {
+        $post = $repository->find('BlogBundle:post');
```

Comment:
Added this section because it wasn't immediately clear to me how to use this bundle :)